sympy/physics/mechanics/body.py

```diff
@@ -224,11 +241,83 @@ def apply_torque(self, vec):
             >>> body = Body('body')
             >>> T = Symbol('T')
             >>> body.apply_torque(T * body.frame.z)
+
+        Notes
+        =====
+
+        If a new torque is applied on body which already has some torque applied on it,
+        then the new torque is added to the previous torque about the body's frame.
+
         """
 
         if not isinstance(vec, Vector):
             raise TypeError("A Vector must be supplied to add torque.")
-        self.loads.append((self.frame, vec))
+        for load in self._loads:
+            if self.frame in load:
+                vec += load[1]
+                self._loads.remove(load)
+                break
+        self._loads.append((self.frame, vec))
+
+    def clear_load(self):
+        """
+        Clears the Body's loads list.
+
+        Example
+        =======
+
+        >>> from sympy.physics.mechanics import Body
+        >>> B = Body('B')
+        >>> force = B.x + B.y
+        >>> B.apply_force(force)
+        >>> B.loads
+        [(B_masscenter, B_frame.x + B_frame.y)]
+        >>> B.clear_load()
+        >>> B.loads
+        []
+
+        """
+
+        self._loads = []
```

Comment:
If `.load` is mutable, this method is not necessary. A rule of thumb to remember, if the implementation of the method is only 1 line, you likely don't need the method. (that often goes for 2 or even 3 lines too)

Reply:
I think it should be immutable.

Reply:
I explained above.